src/controllers/state-controller.js

```diff
@@ -0,0 +1,34 @@
+const States = require('../models/states');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const States = require("../models/states");
```